compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -371,6 +371,54 @@ OMR::ARM64::TreeEvaluator::vdivEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return inlineVectorBinaryOp(node, cg, divOp);
    }
 
+TR::Register *
+OMR::ARM64::TreeEvaluator::vandEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::InstOpCode::Mnemonic andOp;
+   switch(node->getDataType())
+      {
+      case TR::VectorInt8:
+         andOp = TR::InstOpCode::vand16b;
+         break;
+      default:
+         TR_ASSERT(false, "unrecognized vector type %s\n", node->getDataType().toString());
+         return NULL;
+      }
+   return inlineVectorBinaryOp(node, cg, andOp);
+   }
+
+TR::Register *
+OMR::ARM64::TreeEvaluator::vorEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::InstOpCode::Mnemonic orrOp;
+   switch(node->getDataType())
+      {
+      case TR::VectorInt8:
+         orrOp = TR::InstOpCode::vorr2d;
```

Comment:
`vand` and `veor` evaluators use opcodes of type `16b`.  Please define and use `vorr16b` here to make it consistent with other evaluators.